CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## [v1.10.2](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/v1.10.2) (2023-07-17)
+
+[Full Changelog](https://github.com/mongodb/terraform-provider-mongodbatlas/compare/v1.10.1...v1.10.2)
+
+**Bug Fixes**
+
+- INTMDB-941: \[Terraform\] Advanced Cluster Doc is not formatted correctly [\#1326](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1326) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- INTMDB-922: mongodbatlas\_event\_trigger is not updated if config\_match is added [\#1305](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1305) ([andreaangiolillo](https://github.com/andreaangiolillo))
+
+**Internal Improvements**
+
+- INTMDB-938: Fix Online Archive tests [\#1321](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1321) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- INTMDB-935: HELP: mongodbatlas\_online\_archive schedule parameter update causing crashing in terraform apply [\#1320](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1320) ([andreaangiolillo](https://github.com/andreaangiolillo))
```

Comment:
I missed one last thing, sorry. These two lines should be under bug fixes instead of internal improvements. They are in this section because I forgot to add the bug label to the PR 😭

Reply:
sorry only the INTMDB-935: HELP: mongodbatlas\_online\_archive schedule parameter update causing crashing in terraform apply [\#1320](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1320) ([andreaangiolillo](https://github.com/andreaangiolillo))

Reply:
Fix Online Archive tests should remain in internal improvements

Reply:
thanks

Reply:
no worries, updated